src/sentry/models/userrole.py

```diff
@@ -0,0 +1,34 @@
+from typing import FrozenSet
+
+from django.db import models
+
+from sentry.db.models import ArrayField, Model, sane_repr
+
+
+class UserRole(Model):
+    """
+    Roles are applied to administrative users and apply a set of `UserPermission`.
+    """
+
+    __include_in_export__ = True
+
+    name = models.CharField(max_length=32, unique=True)
+    permissions = ArrayField()
```

Comment:
Would we ever want to have permissions in the database and have this also be a `ManyToManyField`?

Reply:
prob not worth it - they're intended to be "custom" (e.g. UserPermission is a charfield) so this kind of just keeps that going. savings also dont seem worth it given the low quantity of rows thatll exist